game_event/templates/game_event/join-event.html

```diff
@@ -3,6 +3,7 @@
 <html>
   <head>
     <link rel="stylesheet" type="text/css" href="{% static 'css/game-event.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/game-events.css' %}">
```

Comment:
I can't see file `static/css/game-events.css` in this PR. Perhaps this line is redundant?